DbService/src/SConscript

```diff
@@ -46,6 +46,8 @@ helper.make_plugins( [ mainlib,
 BINLIBS   = [ mainlib, 'mu2e_DbTables' , 'cetlib', 'cetlib_except', "pq" ]
 helper.make_bin("dbTool",BINLIBS,[])
 
+# turn pywrap.i into a python interface
+helper.make_pywrap (['mu2e_DbTables'])
```

Comment:
Where is the source for make_pywrap?  I don't see it either in Offline or in Muse.

Reply:
[here](https://github.com/Mu2e/Muse/blob/91127678ce9ae6600fa435408548cdd4221099d0/python/mu2e_helper.py#L125) in Muse.  You must have just overlooked it.

Reply:
Thanks.  Master/main confusion in my clone.